nsiqcppstyle_exe.py

```diff
@@ -184,19 +186,21 @@ def main(argv=None):
             elif o == "--var":
                 varMap = GetCustomKeyValueMap(a, "--var="+a)
             elif o == "--ci":
-                ciMode = True
+                console.SetVerbosity(Verbosity.Ci);
+            elif o in ("-q", "--quiet"):
+                console.SetVerbosity(Verbosity.Error);
             elif o == "--noBase":
                 noBase = True
 
-        print title
+        console.CI(title)
         runtimePath = GetRuntimePath()
         sys.path.append(runtimePath)
         if updateNsiqCppStyle:
             try:
-                print "======================================================================================"
+                console.CI(console.Separator)
```

Comment:
`try` should be only for `updateagent` since console can fail for other reasons too.